lib/features/rules_filter_creator/presentation/model/rule_filter_action_arguments.dart

```diff
@@ -0,0 +1,34 @@
+import 'package:equatable/equatable.dart';
+import 'package:model/mailbox/presentation_mailbox.dart';
+import 'package:tmail_ui_user/features/rules_filter_creator/presentation/model/email_rule_filter_action.dart';
+
+class RuleFilterActionArguments with EquatableMixin {
```

Comment:
please separate argument for each action. Thanks

Reply:
I separated it!